File: dataset_reduzido/codenet/Java/Code/s853705214.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int[] nums = new int[n];

        for (int i = 0; i < n; i++) {
            nums[i] = sc.nextInt();
        }

        int min = Integer.MAX_VALUE;
        int countNegatives = 0;
        long sum = 0;
        for (int num : nums) {
            int abs = Math.abs(num);
            sum += abs;
            min = Math.min(min, abs);
            if (num < 0) {
                countNegatives++;
            }
        }

        System.out.println(countNegatives % 2 == 0 ? sum : sum - (min*2));
    }
}

File: output/Llama-2-100/codenet/Java/Python/s853705214.py
def main():
    n = int(input())
    nums = [int(input()) for _ in range(n)]

    min = float('inf')
    count_negatives = 0
    sum = 0
    for num in nums:
        abs = abs(num)
        sum += abs
        min = min(min, abs)
        if num < 0:
            count_negatives += 1

    print(count_negatives % 2 == 0, sum)
